### PR TITLE
DBFT: add dbft unit test

### PR DESCRIPTION
**Pull request by Liaojinghui**
add the unit test for dbft

### CODE REVIEW

**Comment by Liaojinghui** [comment]
@igormcoelho Hey man, i tried my best to migrate this into N3, but there still a few accessible issues, i think you might delete them directly, so i will stop from here.

File: neo-modules.sln [hunk 1]
```diff
@@ -43,6 +43,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SQLiteWallet", "src\SQLiteW
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "StorageDumper", "src\StorageDumper\StorageDumper.csproj", "{938D86EA-0F48-436B-9255-4AD9A8E6B9AC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Neo.Plugins.DBFTPlugin.Tests", "tests\Neo.Plugins.DBFTPlugin.Tests\Neo.Plugins.DBFTPlugin.Tests.csproj", "{CA5D6744-F0C1-4B08-9B14-E202BEC2522C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Neo", "..\neo\src\Neo\Neo.csproj", "{EC0058F3-89ED-4591-BDA3-C8B8A04F3F8F}"
+EndProject
```
Comment (cschuchardt88) [inline neo-modules.sln:49]: Should we use the `nuget` package instead? `github` actions is failing.

```suggestion

```

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

**Comment by shargon** [comment]
@Liaojinghui please give me rights in your branch to change some small things

**Comment by shargon** [comment]
@Liaojinghui ut fails

**Comment by Liaojinghui** [comment]
my bad, this one should be marked as draft, @vncoelho want to build the unit test for dbft, haven't complete

**Comment by Liaojinghui** [comment]
> @shargon my bad, this one should be marked as draft, @vncoelho want to build the unit test for dbft, haven't complete, @vncoelho will do the rest

**Pull request closed without merging**